helmfile.d/values/prometheus-blackbox-exporter-sc.yaml.gotmpl

```diff
@@ -4,6 +4,10 @@ pspEnabled: false
 
 config:
   modules:
+    http_2xx:
+      http:
+        tls_config:
+          insecure_skip_verify: {{ not .Values.global.verifyTls }}
```

Comment:
Should we (is it possible) add a comment here to discuss that it's not recommended, but might be required in some Air-gapped environments.

Reply:
As this is a global variable that is used to change configurations of other services as well I don't see a need to have a comment for it (not here at least), as this should always be set to true in production environments. I added this so that you don't get alerts when `verifyTls=false`, as in this case you have made a conscious choice to not care about verifying TLS.